packages/buefy-next/src/components/switch/Switch.vue

```diff
@@ -88,7 +90,8 @@ export default {
             get() {
                 return this.newValue
             },
-            set(value) {
+            /*String, Number, Boolean, Function, Object, Array, Date*/
+            set(value: string|boolean|Function|object|Array<string|boolean|Function|object|Date>|Date) {
```

Comment:
`number` is missing. `value` can be almost `any` value. I think using `any` is acceptable here.

Reply:
oops😅

Although any may be desired here I get the following linting error when I set `value` to `any`

`93:24  error  Unexpected any. Specify a different type        @typescript-eslint/no-explicit-any`

Do you want to configure es-lint to allow any? Or should I just add `number` to `value`s type?

Reply:
You can disable ESLint rules on a specific line with `eslint-disable-next-line`. For example:
https://github.com/ntohq/buefy-next/blob/86d37c945361bcd741763fd866c3fe4578f6d9ca/packages/buefy-next/src/components/autocomplete/Autocomplete.vue#L155-L156